src/explorer/SubscriptionTreeItem.ts

```diff
@@ -100,6 +104,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
 
         if (!context.advancedCreation) {
             await setDefaultRgAndPlanName(wizardContext, siteStep);
+            wizardContext.newAppInsightsName = await wizardContext.relatedNameTask;
```

Comment:
`relatedNameTask` can technically return undefined. Suggest something like this:
https://github.com/microsoft/vscode-azurefunctions/blob/master/src/tree/SubscriptionTreeItem.ts#L136